language/move-vm/runtime/src/move_vm.rs

```diff
@@ -21,53 +24,74 @@ impl MoveVM {
         }
     }
 
+    pub fn new_session<'r, C: RemoteCache>(&self, remote: &'r C) -> Session<'r, '_, C> {
+        Session {
+            runtime: &self.runtime,
+            data_cache: TransactionDataCache::new(remote, self.runtime.loader()),
+        }
+    }
+}
+
+impl Default for MoveVM {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+pub struct Session<'r, 'l, R> {
```

Comment:
I am not clear about the `Session` story but I think it's good enough for now and we can go out that way!